mod/item.php

```diff
@@ -1,78 +1,76 @@
 <?php
 
-/**
- *
+/*
  * This is the POST destination for most all locally posted
  * text stuff. This function handles status, wall-to-wall status,
  * local comments, and remote coments that are posted on this site
  * (as opposed to being delivered in a feed).
  * Also processed here are posts and comments coming through the
  * statusnet/twitter API.
+ *
  * All of these become an "item" which is our basic unit of
  * information.
+ *
  * Posts that originate externally or do not fall into the above
  * posting categories go through item_store() instead of this function.
- *
  */
 
-require_once('include/crypto.php');
-require_once('include/enotify.php');
-require_once('include/email.php');
-require_once('include/tags.php');
-require_once('include/files.php');
-require_once('include/threads.php');
-require_once('include/text.php');
-require_once('include/items.php');
-require_once('include/Scrape.php');
-require_once('include/diaspora.php');
-require_once('include/Contact.php');
+require_once 'include/crypto.php';
+require_once 'include/enotify.php';
+require_once 'include/email.php';
+require_once 'include/tags.php';
+require_once 'include/files.php';
+require_once 'include/threads.php';
+require_once 'include/text.php';
+require_once 'include/items.php';
+require_once 'include/Scrape.php';
+require_once 'include/diaspora.php';
+require_once 'include/Contact.php';
 
 function item_post(App $a) {
 
-	if((! local_user()) && (! remote_user()) && (! x($_REQUEST,'commenter')))
+	if ((! local_user()) && (! remote_user()) && (! x($_REQUEST,'commenter'))) {
 		return;
+	}
 
 	require_once('include/security.php');
```

Comment:
Oh, using relative paths means that PHP is search the whole `include_path` ini setting. That should be avoided at all costs as it costs performance (see documentation). Better is to e.g. "auto-detect" the base path (like in `boot.php` with `static $base_path = __DIR__;`) and then use it everywhere.